html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> HTML Forms </title>
+</head>
+
+<body>
+
+<h1>Coding Form!</h1>
+
+<form action = 'index.html' method = 'post'></form>
+  <label for='mail'>Email</label>
+  <div>
+    <input type='email' id='mail' name='user_email'>
+  </div>
+    <label for='name'>Why is coding cool?</label>
+
+  <div>
+    <select id="simple" name="simple">
+      <option selected>Select an option</option>
+      <option>Because it's interesting</option>
+      <option>Because it's fun</option>
+      <option>Because it's powerful</option>
+      </select>
+  </div>
+
+  <div>
+    <label for='text'>Favorite Language (choose one)</label>
```

Comment:
This `label` isn't associated with any form control element, so it should just be a `p` element or something